compiler/one-cmds/one-quantize

```diff
@@ -308,10 +316,10 @@ def _quantize(args):
         # input and output path
         if _utils._is_valid_attr(args, 'input_path'):
             circle_quantizer_cmd.append(getattr(args, 'input_path'))
-        tmp_output_path_1 = os.path.join(
+        tmp_weights_fake_quant_path = os.path.join(
             tmpdir,
-            os.path.splitext(os.path.basename(args.input_path))[0]) + '1.circle'
-        circle_quantizer_cmd.append(tmp_output_path_1)
+            os.path.splitext(os.path.basename(args.input_path))[0]) + '.weights_fake_quant.circle'
+        circle_quantizer_cmd.append(tmp_weights_fake_quant_path)
```

Comment:
As you wrote, you renamed this for better readability.
But this explanation is not included in the commit message nor in the title.
(1) it would be better to splti changes to two PRs
(2) please revise commit message and title that explains two changes

Reply:
I've update the commit message and the title. PTAL

Reply:
I think current commit message is not changed, only this PR has changed.

https://github.com/Samsung/ONE/pull/9157/commits/01e2e842fa4a3296fbd03e5fbd4bd2a2ce6ef8ac